statistics/semantic/src/main/java/com/spotify/heroic/statistics/semantic/SemanticMetricBackendReporter.java

```diff
@@ -196,6 +211,16 @@ public void reportWritesDroppedBySize() {
         writesDroppedBySize.inc();
     }
 
+    @Override
+    public void reportGlobalReadDataPoints(long points) {
+        globalReadDataPoints.set(points);
```

Comment:
`reportX` to me means to log or print something i.e. To generate a report. Any reason we're not using the traditional `setGlobalReadDataPoints`?

Reply:
The naming is inline of SemanticMetricBackendReporter as the object name implies.

Reply:
Hmm. OK.